### DISCUSSION TITLE
Range of viscosity values for standard method?

### DISCUSSION THREAD

**Discussion by arjun-mani** [InteractiveComputerGraphics/SPlisHSPlasH#193] [Q&A] [answered]
Hello, 

Thanks very much for your kind and really prompt help with issues. I had a question about the range of values for the standard viscosity method. It seems like 0.01 is the default value in the examples and produces behavior that looks like water. I have tried 0.05 and 0.1 and these tend to produce increasingly viscous behavior; however, trying 0.2 or higher values produces nonsensical behavior (particles jitter or explode). I was just curious, if I wanted to capture a range of viscosities (say water to honey), what range of values should I use?  I am just a bit confused because per my knowledge kinematic viscosity values are typically a lot lower.

Thanks very much in advance.

**Comment by janbender** [comment, marked as answer]
Hi, 

The standard viscosity uses an explicit time integration. Like all explicit methods the integration is only conditionally stable. That means for high viscosity coefficients the simulations tends to get unstable. So if you want to simulate highly-viscous material behavior, I recommend to use the implicit method of Weiler et al. 2018 which is also implemented. To choose the coefficent you can look in tables like here: https://en.wikipedia.org/wiki/Viscosity

Hope that helps!
